### PR TITLE
Add script to pull string changes from thimble on build

### PR DESCRIPTION
**Pull request by gideonthomas**

### CODE REVIEW

**Comment by gideonthomas** [comment]
Related to: https://github.com/mozilla/thimble.mozilla.org/pull/1439

File: scripts/get-file-sha.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+}, function(err, res, body) {
+    "use strict";
+
+    if(!err && res.statusCode === 200) {
+        try {
+            var content = JSON.parse(body);
+            process.stdout.write(content.sha);
+        } catch(e) {
+            console.error("Failed to convert response to JSON with: ", e);
+            process.exit(0);
```
Comment (jbuck) [inline scripts/get-file-sha.js]: This should exit with a non-zero error code if it's a fatal error

File: scripts/pull-new-strings.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        # Base64 encode the new file's contents
+        content="$(base64 $CHANGED_FILE)"
+        body="{ \
+            \"message\": \"Thimble-L10N - Update strings for $locale\", \
+            \"content\": \"$content\", \
+            \"sha\": \"$sha\" \
+        }"
+
+        # Commit the change to the repo
+        curl --location-trusted -X PUT \
```
Comment (jbuck) [inline scripts/pull-new-strings.sh]: Are you sure you need --location-trusted here and not just -L ?

File: scripts/get-file-sha.js (hunk 1 above)
Comment (gideonthomas) [inline scripts/get-file-sha.js]: oh whoops, my bad

**Comment by jbuck** [comment]
This seems reasonable to me

**Comment by jbuck** [comment]
r+ with some nits

**Pull request merged**